mne/io/bti/bti.py

```diff
@@ -554,10 +554,10 @@ def dev_header(x):
             fid.seek(2, 1)
             ch.update({'gain': read_float(fid),
                        'units_per_bit': read_float(fid),
-                       'yaxis_label': read_str(fid, 16),
+                       'yaxis_label': _read_str(fid, 16),
```

Comment:
look we use read_xxx everywhere. I'd stick to the read_str naming for consistency.